mteb/evaluation/evaluators/utils.py

```diff
@@ -53,6 +53,24 @@ def dot_score(a: torch.Tensor, b: torch.Tensor):
     return torch.mm(a, b.transpose(0, 1))
 
 
+def max_sim(a: list, b: list):
+    """Computes the max-similarity max_sim(a[i], b[j]) for all i and j.
+
+    Return:
+        Matrix with res[i][j]  = max_sim(a[i], b[j])
+    """  # noqa: D402
+    a = torch.stack(a)
+    b = torch.stack(b)
```

Comment:
When I run the example in README, I get an error pointing to here:
```
  File "/home/ubuntu/isaac/work/mteb/mteb/evaluation/evaluators/RetrievalEvaluator.py", line 194, in search
    cos_scores = self.score_functions[score_function](
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/isaac/work/mteb/mteb/evaluation/evaluators/utils.py", line 63, in max_sim
    a = torch.stack(a)
        ^^^^^^^^^^^^^^
TypeError: stack(): argument 'tensors' (position 1) must be tuple of Tensors, not numpy.ndarray
```

Reply:
Removing the `score_function` resolves the issue because it is already defined in the `ModelMeta`. Including it again is redundant and unnecessary.